sklearn/utils/estimator_checks.py

```diff
@@ -600,18 +593,6 @@ def check_estimator(estimator=None, generate_only=False, Estimator="deprecated")
     parametrize_with_checks : Pytest specific decorator for parametrizing estimator
         checks.
     """
-
-    if estimator is None and Estimator == "deprecated":
-        msg = "Either estimator or Estimator should be passed to check_estimator."
-        raise ValueError(msg)
-
-    if Estimator != "deprecated":
-        msg = (
-            "'Estimator' was deprecated in favor of 'estimator' in version 1.1 "
-            "and will be removed in version 1.3."
-        )
-        warnings.warn(msg, FutureWarning)
-        estimator = Estimator
     if isinstance(estimator, type):
         msg = (
             "Passing a class was deprecated in version 0.23 "
```

Comment:
I just found this part of the code and the PR with it:
```
    if isinstance(estimator, type):
        msg = (
            "Passing a class was deprecated in version 0.23 "
            "and isn't supported anymore from 0.24."
            "Please pass an instance instead."
        )
        raise TypeError(msg)
```

Can't it be removed as well?